staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest_test.go

```diff
@@ -424,9 +424,9 @@ func TestNumberConversion(t *testing.T) {
 	versionedObjToUpdate := &examplev1.Pod{}
 	schemaReferenceObj := &examplev1.Pod{}
 
-	patchJS := []byte(`{"spec":{"terminationGracePeriodSeconds":42,"activeDeadlineSeconds":120}}`)
+	patchBytes := []byte(`{"spec":{"terminationGracePeriodSeconds":42,"activeDeadlineSeconds":120}}`)
```

Comment:
This line and the next probably don't need to change :)

I mean, it doesn't hurt anything, it just makes the diff a bit bigger--I wouldn't even mention it except we're on the branch and all.

Reply:
However I think it'd be good to add a unit test to this file.